tests/core/test_broker.py

```diff
@@ -22,6 +22,7 @@
 from rasa.core.brokers.kafka import KafkaEventBroker
 from rasa.core.brokers.pika import PikaEventBroker, DEFAULT_QUEUE_NAME
 from rasa.core.brokers.sql import SQLEventBroker
+from rasa.shared.exceptions import RasaException
```

Comment:
i just noticed there is actually an existing import from the same module in L27 - could you add the `, RasaException` to L27 and remove duplicate import in L25?

Reply:
ok